instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/aiopg_integration.py

```diff
@@ -109,12 +109,13 @@ async def traced_execution(
             try:
                 result = await query_method(*args, **kwargs)
                 if span.is_recording():
-                    span.set_status(Status(StatusCanonicalCode.OK))
+                    # TODO: Remove setting status in instrumentation
```

Comment:
```suggestion
                    # TODO: Remove setting OK status in instrumentation
```

Reply:
That's not exactly accurate. The TODO is removing any `set_status` in instrumentations that pertain to `OK` or `UNSET`. I should probably name it to `remove setting non-ERROR status in instrumentation`.

Reply:
Update: I made the changes and will be just removing the TODOs.